entry_points.txt

```diff
@@ -14,6 +14,7 @@ analysis_profiles = cfme.configure.configuration.analysis_profile:AnalysisProfil
 ansible_credentials = cfme.ansible.credentials:CredentialsCollection
 ansible_playbooks = cfme.ansible.playbooks:PlaybooksCollection
 ansible_repositories = cfme.ansible.repositories:RepositoryCollection
+ansible_tower = cfme.infrastructure.config_management:AnsibleTowersCollection
```

Comment:
This needs to be plural.
May be :
config_manager_ansible_tower_providers = cfme.infrastructure.config_management:AnsibleTowersCollection